### DISCUSSION TITLE
is it possible to have video in a widget?

### DISCUSSION THREAD

**Discussion by bkjohnson** [elkowar/eww#464] [Q&A]
I'm wondering if it's possible to have a widget that shows live video, like from a webcam.

**Comment by elkowar** [comment]
Not currently, no
What would be the usecase here, especially for life video?

**Comment by bkjohnson** [comment]
I was imagining a widget that could be styled & positioned so that it could be embedded in a screen share type of situation. This could allow the presenter to still be viewed while viewing the shared screen at full screen.

**Reply by pokety** [reply]
https://github.com/maykbrito/mini-video-me
